src/objc/TuriCreateObjC.m

```diff
@@ -25,6 +25,7 @@ static bool handleError(tc_error* error_ptr, NSError** error) {
                                          code:MLModelErrorGeneric
                                      userInfo:userInfo];
         }
+        assert(false);
```

Comment:
Do you want to keep this line? Is the return statement on the line below just to make the compiler happy?

Reply:
I added it to debug; the return statement is probably the correct behavior here. I'll remove this line.